src/main/resources/db/changelog/changelog-1.yaml

```diff
@@ -0,0 +1,729 @@
+databaseChangeLog:  
+  -  changeSet:
+      id: 1
+      author: aleksei
```

Comment:
Could you pls change author name into email

Reply:
fixed